osrf/vrx
# Station keeping practice worlds

How to test it?

Launch the simulation:
```
ros2 launch vrx_gz competition.launch.py world:=practice_2022_stationkeeping0_task
```

Launch the teleop node:
```
ros2 launch vrx_gz usv_joy_teleop.py
```

Now, you can move around and check the goal and errors by echoing the topics:
```
/vrx/stationkeeping/goal
/vrx/stationkeeping/mean_pose_error
/vrx/stationkeeping/pose_error
```
